CHANGELOG.md

```diff
@@ -8,14 +8,26 @@ Mapbox welcomes participation and contributions from everyone.
 
 * `BasicCameraAnimator` now keeps animators alive without the user storing the animator. ([#646](https://github.com/mapbox/mapbox-maps-ios/pull/646/))
 * Experimental style APIs are now marked with `@_spi(Experimental)` and the previously used underscore prefixes have been removed. In order to access these methods, use `@_spi(Experimental)` to annotate the import statement for MapboxMaps. ([#680](https://github.com/mapbox/mapbox-maps-ios/pull/680)) 
+* Refine RenderedQueryOptions.filter for Swift ([#2150](https://github.com/mapbox/mapbox-gl-native-internal/pull/2150))
+* Updated Objective-C APIs to use NS_REFINED_FOR_SWIFT more consistently. ([#2157](https://github.com/mapbox/mapbox-gl-native-internal/pull/2157))
+* Refine `OfflineRegionGeometryDefinition.geometry` for Swift ([#2175](https://github.com/mapbox/mapbox-gl-native-internal/pull/2175))
```

Comment:
I think we should include the associated Turf refinements in this PR and simplify the change log to just say that the types of the properties have changed